glib/src/gstring.rs

```diff
@@ -1757,6 +1758,90 @@ impl crate::value::ToValue for Vec<GString> {
 impl_from_glib_container_as_vec_string!(GString, *const c_char);
 impl_from_glib_container_as_vec_string!(GString, *mut c_char);
 
+// rustdoc-stripper-ignore-next
+/// A trait to accept both <code>&[str]</code> or <code>&[GStr]</code> as an argument.
```

Comment:
Why `<code>` instead of just backticks?

Reply:
This looks confusing, these are not arrays. The `[str]` is a link and the `&` is not. So it will look like <code>&[str](https://doc.rust-lang.org/std/primitive.str.html)</code>

Reply:
Oh I see, that's why.